ModulemdTranslationHelpers/Utils.py

```diff
@@ -58,3 +63,84 @@ def get_translation_catalog_from_index(index, project_name):
         catalog.add(translatable_string, locations=locations)
 
     return catalog
+
+
+def get_modulemd_translations_from_catalog(catalogs, index):
+    # Dictionary `translations` contains information from catalog like:
+    # Key: (module_name, stream_name)
+    # Value: Translation object containing TranslationEntry of various locales
+    translations = dict()
+
+    now = datetime.utcnow()
+    modified = int("%04d%02d%02d%02d%02d%02d" % (
+        now.date().year,
+        now.date().month,
+        now.date().day,
+        now.time().hour,
+        now.time().minute,
+        now.time().second
+    ))
```

Comment:
This assignment can be simplified to `modified = int(now.strftime("%Y%m%d%H%M%S"))`